src/components/toastNotificationsList/toastNotificationsList.js

```diff
@@ -21,46 +25,51 @@ class ToastNotificationsList extends React.Component {
   };
 
   render() {
-    const { toasts, paused } = this.props;
+    const { toasts, timeout } = this.props;
 
     return (
-      <ToastNotificationList>
-        {toasts &&
-          toasts.map((toast, index) => {
-            if (!toast.removed) {
-              return (
-                <TimedToastNotification
-                  key={helpers.generateId('key')}
-                  toastIndex={index}
-                  type={toast.alertType}
-                  paused={paused}
-                  onDismiss={() => this.onDismiss(toast)}
-                  onMouseEnter={this.onHover}
-                  onMouseLeave={this.onLeave}
-                >
-                  <span>
-                    <strong>{toast.header}</strong> &nbsp;
-                    {toast.message}
-                  </span>
-                </TimedToastNotification>
-              );
-            }
-
-            return null;
-          })}
-      </ToastNotificationList>
+      <AlertGroup isToast className="quipucords-toast-notifications__alert-group">
+        {toasts?.map(toast => {
+          if (!toast.removed) {
```

Comment:
required:
we need to consume the redux state with the updated toasts. we can leverage the PF timeout attribute's use of `undefined`/`boolean`/`number`

```
...removed ){
return (
  <Alert
    ...
    timeout={!toast.paused && timeout}
```

this update combined with the requested reducer updates should get the toast pause/resume working again. you can take look by setting up a bunch of scans in a row from the scans view